setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# This file is part of ctrl_mpexec.
```

Comment:
Should be ctrl_bps instead of ctrl_mpexec.

Reply:
yes. Copy and paste.